docs/public-networks/reference/cli/subcommands.md

```diff
@@ -57,6 +57,8 @@ You can specify the starting index of the block range to import with `--start-bl
 
 You can specify the ending index (exclusive) of the block range to import with `--end-block`. If omitted, all blocks after the start block will be imported.
 
+You can specify multiple `--from` arguments. This can be useful when blocks have been exported over time to multiple files. If multiple files are provided they will be read in the order specified in the command.
```

Comment:
To specify multiple `--from` arguments, do you separate the arguments with commas or do you specify the `--from` option multiple times?

Reply:
@alexandratran you specify it multiple times. I did wonder about adding more than 1 `--from` argument into the example in the docs, if you think that would be useful?

Reply:
Yeah, that would be helpful!

Reply:
Done :) See new commit 2bf1add6c18c47ae820d6fed0b94eec8945697b9